package.json

```diff
@@ -3,14 +3,39 @@
   "repo": "johntron/superagent-no-cache",
   "description": "Plugin for visionmedia/superagent that adds headers to all requests that prevents caching",
   "version": "0.0.1",
-  "keywords": [],
-  "dependencies": {
-      "component/ie": "0.0.1"
+  "keywords": [
+    "superagent",
+    "cache",
+    "request",
+    "AJAX"
+  ],
+  "author": "John Syrinek <john.syrinek@gmail.com>",
+  "contributors": [
+    "Trent Oswald <trentoswald@therebelrobot.com> [@therebelrobot] (http://therebelrobot.com)"
+  ],
+  "bugs": {
+    "url": "https://github.com/johntron/superagent-no-cache/issues"
   },
-  "development": {},
+  "homepage": "https://github.com/johntron/superagent-no-cache",
   "license": "MIT",
   "main": "index.js",
-  "scripts": [
-    "index.js"
-  ]
+  "scripts": {
+    "test": "./node_modules/standard/bin/cmd.js && ./node_modules/istanbul/lib/cli.js cover _mocha --"
```

Comment:
The second command (running the tests) doesn't execute if ferross/standard exits with a failure code (seems to do this when there are any formatting issues. Maybe we should use:
```shell
./node_modules/standard/bin/cmd.js; ./node_modules/istanbul/lib/cli.js cover _mocha --
```

If this was intended to run in CI instead of from the CLI, then maybe the `&&` makes sense.